src/ansi-c/expr2c.cpp

```diff
@@ -2329,23 +2329,24 @@ std::string expr2ct::convert_rox(const exprt &src, unsigned precedence)
     mod_exprt(rotate_expr.distance(), width_expr);
   // Now put the pieces together
   // width(AAAA) - (n % width(AAAA))
-  const auto complement_expr = minus_exprt(width_expr, distance_modulo_width);
+  const auto complement_width_expr =
+    minus_exprt(width_expr, distance_modulo_width);
   // lhs and rhs components defined according to rol/ror
   exprt lhs_expr;
   exprt rhs_expr;
   if(src.id() == ID_rol)
   {
     // AAAA << (n % width(AAAA))
     lhs_expr = shl_exprt(op0, distance_modulo_width);
-    // AAAA >> complement_expr
-    rhs_expr = ashr_exprt(op0, complement_expr);
+    // AAAA >> complement_width_expr
+    rhs_expr = ashr_exprt(op0, complement_width_expr);
   }
   else if(src.id() == ID_ror)
   {
     // AAAA >> (n % width(AAAA))
     lhs_expr = ashr_exprt(op0, distance_modulo_width);
-    // AAAA << complement_expr
-    rhs_expr = shl_exprt(op0, complement_expr);
+    // AAAA << complement_width_expr
+    rhs_expr = shl_exprt(op0, complement_width_expr);
```

Comment:
:pick: These changes appear to have been squashed into the wrong commit.

Reply:
Fixed.